tests/functional/spec/features/new_calculated_summary_repeating_section.spec.js

```diff
@@ -203,8 +205,22 @@ describe("Feature: Calculated Summary Repeating Section", () => {
       expect($(NumberTotalPlaybackPage.sixthNumberAnswer()).getText()).to.contain("45.67");
     });
 
-    it("Given I complete every calculated summary, When I go to a page with calculated summary piping, Then I should the see the piped calculated summary total for each summary", () => {
+    it("Given I have a calculated summary total that is used as a placeholder in another calculated summary, When I get to the calculated summary page displaying the placeholder, Then I should see the correct total", () => {
       $(NumberTotalPlaybackPage.submit()).click();
+      expect(browser.getUrl()).to.contain(BreakdownPage.pageName);
+      $(BreakdownPage.answer1()).setValue(100.0);
+      $(BreakdownPage.answer2()).setValue(24.58);
+      $(BreakdownPage.submit()).click();
+      expect(browser.getUrl()).to.contain(SecondCurrencyTotalPlaybackPage.pageName);
+      expect($(SecondCurrencyTotalPlaybackPage.calculatedSummaryTitle()).getText()).to.contain(
+        "We calculate the total of number values entered to be £124.58. Is this correct?"
+      );
+      expect($("body").getText()).to.have.string("Enter two values that add up to the previous calculated summary total of £124.58");
```

Comment:
Not ideal and should really have its own method. Happy to merge since it is the first one for calculated summary. Can look to introduce a proper method if and when we have multiple. Just need to mindful so we can avoid this being overdone everywhere.